BuildAndTest.proj

```diff
@@ -50,6 +55,10 @@
              Targets="Rebuild"/>
   </Target>
 
+  <Target Name="Sign" Condition="'$(OfficialBuild)' == 'true'">
+    <Exec Command="$(NuGetPackageRoot)\RoslynTools.Microsoft.SignTool\$(RoslynToolsMicrosoftSignToolVersion)\tools\SignTool.exe $(SignRoslynArgs) &quot;$(BinariesPath)&quot;" WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
This matches the approach used by Roslyn repo: See https://github.com/dotnet/roslyn/blob/78f5f50175a051b5bf621e6f785ee1762eccb71b/src/Tools/MicroBuild/Build.proj#L36 and https://github.com/dotnet/roslyn/blob/78f5f50175a051b5bf621e6f785ee1762eccb71b/build/config/SignToolData.json and